dotcom-rendering/src/amp/lib/srcset-utils.ts

```diff
@@ -30,12 +31,16 @@ const removeDuplicateWidth = (items: SrcSetItem[]): SrcSetItem[] => {
 	return packet.items;
 };
 
+const isSrcSet = (
+	item: SrcSetItem | undefined,
+): item is NonNullable<SrcSetItem> => !isUndefined(item);
```

Comment:
This feels like it could live in libs: https://github.com/guardian/csnx/pull/313